docs/learn/learn-async-backing.md

```diff
@@ -7,103 +7,179 @@ keywords: [parachains, slots, backing, parablock]
 slug: ../learn-async-backing
 ---
 
+import RPC from "./../../components/RPC-Connection";
+
+:::info Learn about Parachain Consensus
+
+To fully follow the material on this page, it is recommended to be familiar with the primary stages
+of the [Parachain Protocol](./learn-parachains-protocol.md).
+
+:::
+
+## Synchronous Backing
+
+Before diving into asynchronous backing it is important to understand what synchronous backing is
+and what its main limitations are.
+
+:::info What is backing?
+
+**Backing** refers to the process in which a parablock is verified by a subset of validators who
+determine the blocks validity. It is an important step in the validation process for parablocks, as
+it is the first line of defense in ensuring valid state transitions are taking place. Validators who
+back the parablock put their stake at risk, and if the block is found to be malicious, those
+validators are slashed.
+
+:::
+
+In synchronous backing, parablock validation is tightly coupled to the relay chain's progression on
+a one-to-one basis. Every parablock must be generated and backed within a relay-chain block
+(six-second window), and (if successfully backed) it will be included in a relay-chain block (often
+referred to as the **parent** block, as the parablock anchors itself to it) after an additional six
+seconds. Thus, a parablock can be produced every 12 seconds because a new parablock can be produced
+after the inclusion of the previous one.
+
+The **contextual execution** of new parablocks are parablock ancestors included in the relay chain
+after 12 seconds, where contextual execution refers to the context built by included parablock
+ancestors used by the parachains to generate new parablocks.
+
+:::info It is important to note the clarification below
+
+- Not the entire parablock, but the **paraheader** is placed on the parent block on the relay chain.
+- The relay chain does not access the entire state of a parachain but only the values that changed
+  during that block and the merkelized hashes of the unchanged values.
```

Comment:
Not just the headers of the parablock, but a bit more info goes into the relaychain and the formal term for it is candidate receipt https://medium.com/polkadot-network/the-path-of-a-parachain-block-47d05765d7a

```suggestion
- Not the entire parablock, but the **candidate receipt** consisting of the hash of the parablock, state roots and ID info is placed on the parent block on the relay chain.
- The relay chain does not access the entire state of a parachain but only the values that changed
  during that block and the merkelized hashes of the unchanged values.
  
```